### ISSUE TITLE
#not an issue: This version is much better than Special K

### ISSUE THREAD

**Issue by r3WiNd3r** [soupstream/EldenRingStutterFix#10]
Much better in terms of compatibility with mods, reshade and many other things compared to Special K (tried special K and the game corrupted stating that my save is corrupt, had to completely delete the game and verify files)

Sad to see the updates stopping, but this seems to still be working even with the latest version of the game so thank you very much for the efforts and I will keep using and enjoying this until fromsoft decides to assign at least one half part time intern dev to sort out the performance issues on the lazy PC Port.

Again, thank you for this and for still keeping the repo up.

**Comment by soupstream** [comment]
Yeah I've seen some people saying they've had issues with Special K, so I guess this mod is still useful as simpler fallback. I might update it if another method of improving performance comes to my attention, but otherwise it has all the fixes I wanted to try and it should work with all future versions of Elden Ring.

**Comment by Ryusennin** [comment]
Except for a handful of cases (Nier Automata), I'm not a big fan of SpecialK either. In Elden Ring, while the zone stuttering (slowmo/acceleration) is improved, it introduces constant micro-stuttering which makes the performance globally worse.

Your own fix is simpler to install/setup and makes a better job at smoothing out the framerate.

**Comment by spigias** [comment]
Special K causes issues for me too, this fix just works and is so simple, please keep updating it
Also for me this fix doesn't improve stuttering, instead it give me better fps and fixes the dips. stuttering is fixed(for me) by disabling the MS Enumarator from device manager.
Keep up the good work.

**Comment by spigias** [comment]
> it introduces constant micro-stuttering which makes the performance globally worse.

THIS!
